Jellyfin.Api/Helpers/RequestHelpers.cs

```diff
@@ -55,6 +55,38 @@ public static (string, SortOrder)[] GetOrderBy(IReadOnlyList<string> sortBy, IRe
             return result;
         }
 
+        /// <summary>
+        /// Checks if the user can access a user.
+        /// </summary>
+        /// <param name="claimsPrincipal">The <see cref="ClaimsPrincipal"/> for the current request.</param>
+        /// <param name="userId">The user id.</param>
+        /// <returns>A <see cref="bool"/> whether the user can update the entry.</returns>
+        internal static Guid GetUserId(ClaimsPrincipal claimsPrincipal, Guid? userId)
```

Comment:
I don't really understand this method tbh. If a user is trying to access another user's items, change their info or anything else, it should throw a Forbidden, not default to authenticated user.

Reply:
I went back and forth on whether it should return forbidden, will update in a few days